template/static/styles/site.cerulean.css

```diff
@@ -6529,6 +6529,130 @@ button.close {
     display: none !important;
   }
 }
+.navbar {
```

Comment:
presumably all these static files came from bootswatch and I should undo all the changes?